.github/workflows/deploy-with-fastlane.yaml

```diff
@@ -0,0 +1,155 @@
+name: Publish iOS and Android release
+
+on:
+  release:
+    types: [published]
+  push:
+    tags:
+      - "v*"
+      - "v*-rc.*"
+      - "*.*.*"
+
+env:
+  FLUTTER_CHANNEL: "stable"
+  FLUTTER_VERSION: "3.32.8"
+  RUBY_VERSION: "3.4.5" #"3.2.2"
+
+jobs:
+  # build_ios:
+  #   name: Build iOS
+  #   # See https://docs.github.com/en/actions/using-github-hosted-runners/about-larger-runners/about-larger-runners#about-macos-larger-runners
+  #   runs-on: macos-latest
+  #   timeout-minutes: 20
+  #   steps:
+  #     - name: Checkout
+  #       uses: actions/checkout@v4
+
+  #     - name: Verify macOS version
+  #       run: |
+  #         sw_vers
+
+  #     # - name: Set up Xcode
+  #     #   uses: maxim-lobanov/setup-xcode@v1
```

Comment:
Why are we having these commented blocks

Reply:
That's for the IOS setup, but we can't use that now because GitHub action runtime for IOS is not up to 15 which we need to build without errors, but its rather at 14.8, why I left the setup for when we unify.

Reply:
@nfebe Please kindly review so we resolve this